contracts/UniswapV2Router02.sol

```diff
@@ -0,0 +1,280 @@
+pragma solidity =0.6.6;
+
+import '@uniswap/v2-core/contracts/interfaces/IUniswapV2Factory.sol';
+import '@uniswap/lib/contracts/libraries/TransferHelper.sol';
+
+import './libraries/UniswapV2Library.sol';
+import './interfaces/IUniswapV2Router01.sol';
+import './interfaces/IERC20.sol';
+import './interfaces/IWETH.sol';
+
+contract UniswapV2Router02 is IUniswapV2Router01 {
```

Comment:
are there any methods in the router01 that are unchanged? can we inherit from the router01?

Reply:
per offline, not so easy because we didn't make the functions in 01 `virtual`